cmd/minikube/cmd/start.go

```diff
@@ -1097,6 +1097,13 @@ func validateFlags(cmd *cobra.Command, drvName string) {
 		}
 	}
 
+	if driver.IsSSH(drvName) {
+		sshIPAddress := viper.GetString(sshIPAddress)
+		if sshIPAddress == "" {
```

Comment:
how about adding a validate IP to be a valid ip,
so if user enters a wrong format for the IP we fail with exit.Usage ?

Reply:
I was going to validate the whole SSH connection later, but can check the IP address now